ProcessMaker/processmaker
# Implement UI for Displaying Wizard Templates

This PR introduces the UI for displaying Wizard Templates. Please note that hardcoded wizard template data is currently in place; once the database and backend APIs are developed, this hardcoded data will be replaced with actual data.

## Solution

- Expanded the TemplateSearch component to utilize default template and wizard template styling.
- Implemented UI styling for displaying wizard templates.

## How to Test

1. Navigate to Processes in the Top Menu.
2. Select 'Wizard Templates.'
3. Confirm that the mock data for wizard templates is displayed as laid out in the mockups.

## Related Tickets & Packages
- [FOUR-11916](https://processmaker.atlassian.net/browse/FOUR-11916)

ci:next

## Code Review Checklist


[FOUR-11916]: https://processmaker.atlassian.net/browse/FOUR-11916?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
